src/components/AppNavigation/CalendarList/CalendarListItemSharingSearch.vue

```diff
@@ -195,6 +195,13 @@ export default {
 				return []
 			}
 
+			if (results.data.ocs.meta.status === 'failure') {
+				return []
+			}
+			if (!Array.isArray(results.data.ocs.data.circles)) {
```

Comment:
What else can it return?

Reply:
![F782E963-9E73-47F0-A092-708045A01DFB](https://user-images.githubusercontent.com/1250540/75254182-3f07fb80-57e0-11ea-9fe0-e0a4b783d005.png)

This endpoint provides plenty of things, including groups, addresses for remote sharing, etc. etc.

So it's just some sanity check, so we don't fail with some can not call filter on undefined error.